src/components/Cluster/ClusterDetail/ClusterApps.js

```diff
@@ -10,7 +10,7 @@ import { AppCatalogRoutes } from 'shared/constants/routes';
 import Button from 'UI/Button';
 import ClusterDetailPreinstalledApp from 'UI/ClusterDetailPreinstalledApp';
 
-import AppDetailsModal from './AppDetailsModal';
+import AppDetailsModal from './AppDetailsModal/AppDetailsModal';
```

Comment:
I think it would be sensible to rename `AppDetailsModal.js` to `index.js`, to be able to import it like before:
`import AppDetailsModal from './AppDetailsModal'`

Reply:
I'd be ok with that, but we agreed at some point to stop doing it: https://github.com/giantswarm/fmt/blob/master/javascript/general_style_guide.md#naming

Reply:
I was unaware of that. A workaround would be creating an `index.js` file that just exports `AppDetailsModal` 😅

Reply:
Hmm, not a big concern for me right now. I'd rather have less files and noise to be honest.